### ISSUE TITLE
react schedule-: Memory leak while building project

### ISSUE THREAD

**Issue by Jayrajrodage** [syncfusion/ej2-react-ui-components#144]
Code 
![Screenshot 2024-05-13 154606](https://github.com/syncfusion/ej2-react-ui-components/assets/106481127/66721f8c-b467-42e6-818a-0746f1201d74)
Error
![Screenshot 2024-05-13 154801](https://github.com/syncfusion/ej2-react-ui-components/assets/106481127/0e1310ef-18e6-46fd-8e18-bc9235a952a1)
Package json
![Screenshot 2024-05-13 154814](https://github.com/syncfusion/ej2-react-ui-components/assets/106481127/22688979-124c-4a33-96f3-4a5293971f55)

**Comment by AshokkumarKaruppasamy** [comment]
Hi Jayrajrodage,

We have checked the issues based on the information you shared, but we cannot reproduce the memory leak issue you mentioned. For your convenience, we have included a sample and screenshot below. We have followed some links on how to find the bundle gzip size in a React-Vite project, but we cannot replicate the issues you mentioned in the screenshot. Therefore, we kindly request that you share the steps to reproduce the memory leak issue and provide details on the command to run, so we can understand and resolve the problem.

Link: https://stackoverflow.com/questions/75746767/is-there-any-bundle-analyzer-for-vite

![image](https://github.com/syncfusion/ej2-react-ui-components/assets/107187636/b0303155-7f55-4f5d-a0a9-582d70836cb6)

Sample: [vite-project.zip](https://github.com/user-attachments/files/15887133/vite-project.zip)

Regards,
Ashok
